crates/weaver_forge/README.md

```diff
@@ -579,6 +579,7 @@ comment_formats:           # optional
 
     # Fields specific to 'markdown' format
     escape_backslashes: <bool>            # Whether to escape backslashes in markdown (default: false).
+    escape_square_brackets: <bool>        # Whether to escape square brackets in markdown (default: false).
```

Comment:
Since we have brackets already in the conventions, what is happening there today? And then, shouldn't this be `true` by default, as we want to keep the brackets to match the source data? 🤔

Reply:
I agree. It’s probably better to have this escaping enabled by default. I will make this change before merging. Thanks